mysql-test/suite/multi_source/replicate_rewrite_db_dynamic.test

```diff
@@ -0,0 +1,33 @@
+# Test multi-source dynamically setting of replication filter "replicate_rewrite_db"
+--source include/not_embedded.inc
+--source include/have_innodb.inc
+
+--connect (gandalf_master1,127.0.0.1,root,,,$SERVER_MYPORT_1)
```

Comment:
For the multisource test, you can reference the `mdev-8874.test` file (in multi_source/) because it sets master-specific filters using multiple servers. A key difference though is that `mdev-8874` uses `mdev-8874.cnf` to set the replication filters, and yours should be specified dynamically (as yours already does :))

Reply:
Ok solved